build_runner/lib/src/entrypoint/base_command.dart

```diff
@@ -77,7 +77,10 @@ abstract class BuildRunnerCommand extends Command<int> {
           help: 'Build with release mode defaults for builders.')
       ..addMultiOption(defineOption,
           splitCommas: false,
-          help: 'Sets the global `options` config for a builder by key.');
+          help: 'Sets the global `options` config for a builder by key.')
+      ..addOption(logPerformanceOption,
```

Comment:
[nit] maybe position next to the `trackPerformanceOption` instead of at the end?

Also make a not either here or with the other option that this implies the other.

Reply:
doen